offices/_single.html

```diff
@@ -13,7 +13,7 @@
 
     {% from "post-macros.html" import activity_snippets as activity_snippets with context %}
     {% import "contact-macro.html" as contact_macro %}
-    {% set activities_feed = activity_snippets( office.tags if office.tags ) %}
+    {% set activities_feed = activity_snippets( office.tags if office.tags, include_date_flag=true) %}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/e7de56fa93bd3faf587cc41a42b249154759f12d